src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandom() {
+        Figure[] figure = {
+                new Circle(random.nextDouble(), colorSupplier.getRandomColor()),
+                new RightTriangle(random.nextDouble(),
+                        random.nextDouble(), colorSupplier.getRandomColor()),
+                new Rectangle(random.nextDouble(),
+                        random.nextDouble(), colorSupplier.getRandomColor()),
+                new Square(random.nextDouble(), colorSupplier.getRandomColor()),
+                new IsoscelesTrapezoid(random.nextDouble(),
+                        random.nextDouble(), random.nextDouble(), colorSupplier.getRandomColor())
+        };
+        return figure[random.nextInt(figure.length)];
+    }
```

Comment:
do we need this array?
can we create a figure only after we know what figure should we create? (switch case)